routing/localchans/manager.go

```diff
@@ -155,6 +167,30 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 					"not yet confirmed",
 				))
 
+		case createMissingEdge:
```

Comment:
Let's create a warning log here so we can identify this case next time?

Reply:
Done